src/content/app/genome-browser/state/browser-general/browserGeneralSlice.ts

```diff
@@ -0,0 +1,301 @@
+  reducers: {
+    loadBrowserGeneralState(state) {
+      const activeGenomeId = browserStorageService.getActiveGenomeId();
+      const activeEnsObjectIds = browserStorageService.getActiveEnsObjectIds();
+      const trackStates = browserStorageService.getTrackStates();
+      const chrLocations = browserStorageService.getChrLocation();
+
+      state.activeGenomeId = activeGenomeId;
+      state.activeEnsObjectIds = activeEnsObjectIds;
+      state.trackStates = trackStates;
+      state.chrLocations = chrLocations;
+    },
+    setActiveGenomeId(state, action: PayloadAction<string>) {
+      const genomeId = action.payload;
+      state.activeGenomeId = genomeId;
+    },
+    setGeneralDataFromUrl(state, action: PayloadAction<ParsedUrlPayload>) {
+      const { activeGenomeId, activeEnsObjectId } = action.payload;
+
+      state.activeGenomeId = activeGenomeId;
+      if (activeEnsObjectId) {
+        state.activeEnsObjectIds[activeGenomeId] = activeEnsObjectId;
+      }
+    },
+    updateBrowserActiveEnsObjectIds(
+      state,
+      action: PayloadAction<{ [objectId: string]: string }>
+    ) {
+      state.activeEnsObjectIds = action.payload;
+    },
+    updateTrackStates(state, action: PayloadAction<BrowserTrackStates>) {
+      state.trackStates = Object.assign(state.trackStates, action.payload);
+    },
+    deleteGenomeGeneralData(state, action: PayloadAction<string>) {
+      const genomeIdToRemove = action.payload;
+      const activeGenomeId = state.activeGenomeId;
+
+      delete state.chrLocations[genomeIdToRemove];
+      delete state.actualChrLocations[genomeIdToRemove];
+
+      if (activeGenomeId === genomeIdToRemove) {
+        state.activeGenomeId = null;
+        delete state.activeEnsObjectIds[activeGenomeId];
+      }
+    },
+    setLocationDataFromUrl(state, action: PayloadAction<ParsedUrlPayload>) {
+      const { activeGenomeId, chrLocation } = action.payload;
+      if (chrLocation) {
+        state.chrLocations[activeGenomeId] = chrLocation;
+      } else {
+        delete state.chrLocations[activeGenomeId];
+      }
+    },
+    updateChrLocation(state, action: PayloadAction<ChrLocations>) {
+      state.chrLocations = Object.assign(state.chrLocations, action.payload);
+    },
+    updateActualChrLocation(state, action: PayloadAction<ChrLocation>) {
+      const activeGenomeId = state.activeGenomeId;
+      if (activeGenomeId) {
+        state.actualChrLocations = Object.assign(state.actualChrLocations, {
+          [activeGenomeId]: action.payload
+        });
+      }
+    },
+    toggleRegionEditorActive(state, action: PayloadAction<boolean>) {
+      state.regionEditorActive = action.payload;
+    },
+    toggleRegionFieldActive(state, action: PayloadAction<boolean>) {
+      state.regionFieldActive = action.payload;
+    },
+    updateDefaultPositionFlag(state, action: PayloadAction<boolean>) {
+      state.isObjectInDefaultPosition = action.payload;
+    }
```

Comment:
Could you remind me where, how, and why this action is used? I searched for it in the repo, but could only find its declaration and subsequent consumption of the action. Is it possible that we aren't using it? I hope we are not.

Reply:
It's not used at the moment. Do you think we should completely remove `isFocusObjectInDefaultPosition`?

Reply:
If we can, yes.